modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -695,42 +870,68 @@ curl -s \
   "http://localhost:8081/subjects/sensor-value/versions/1?permanent=true" \
   | jq .
 ```
-
 --
 Python::
 +
 --
-
 ```python
 res = requests.delete(f'{base_uri}/subjects/sensor-value/versions/1').json()
 pretty(res)
 payload = { 'permanent' : 'true' }
 res = requests.delete(f'{base_uri}/subjects/sensor-value/versions/1', params=payload).json()
 pretty(res)
 ```
-
 --
 ====
 
 If the requests are successful, each request returns the version ID of the deleted schema:
```

Comment:
```suggestion
If the requests are successful, each request returns the version ID of the deleted schema:
```

I think this should omit the "If the requests are successful..."

Each time I read this line throughout the PR, I thought, what if it is _not_ successful? It suggests to me that there is some likelihood of it not being successful. Your call, but I'd remove that preface from each of the statements in which it appears throughout.